homeassistant/const.py

```diff
@@ -98,6 +98,7 @@
 CONF_MONITORED_CONDITIONS = 'monitored_conditions'
 CONF_MONITORED_VARIABLES = 'monitored_variables'
 CONF_NAME = 'name'
+CONF_OFF_DELAY = 'off_delay'
```

Comment:
Not sure if this changed since I last contributed more to hass, but shouldn't `homeassistant/const.py` only be for constants that are used many times across platforms?

Reply:
The idea was to use this also in binary_sensor/rfxtrx.py

Reply:
Let's not put this in `homeassistant/const.py` but keep it local.